### PR TITLE
Install Metallb for single cluster too

### PR DESCRIPTION
**Pull request by gargnupur**

### CODE REVIEW

File: common/scripts/kind_provisioner.sh [hunk 1]
```diff
@@ -155,6 +155,9 @@ EOF
   if [[ -n ${METRICS_SERVER_CONFIG_DIR} ]]; then
     kubectl apply -f "${METRICS_SERVER_CONFIG_DIR}"
   fi
+
+  # Install Metallb 
+  install_metallb ""
```
Comment (stevenctl) [inline common/scripts/kind_provisioner.sh:160]: slight nit: Do we need to pass `""`? Maybe we can make sure it's set in the `install_metallb` func with `if -z`.

**Review by stevenctl — APPROVED** [review]
This was tested as it's written now - if you want to merge that's fine just remove the hold.

**Comment by gargnupur** [comment]
> This was tested as it's written now - if you want to merge that's fine just remove the hold.

Yeah I would leave it as is now.. and we can improve it after

**Review by stevenctl — CHANGES_REQUESTED** [review]
This PR needs to be in istio/common-files

**Comment by gargnupur** [comment]
> This PR needs to be in istio/common-files

Thanks for letting me know.. Added https://github.com/istio/common-files/pull/343

**Pull request merged**